recipes/lora_finetune_single_device.py

```diff
@@ -42,13 +43,22 @@ class LoRAFinetuneRecipeSingleDevice(FTRecipeInterface):
     for single GPU training. Training on CPU is not supported.
 
     Features:
-        - Activation Checkpointing. This can be controlled using the ``activation_checkpointing``
+        - Activation Checkpointing. This can be controlled using the ``enable_activation_checkpointing``
```

Comment:
oh...but today the recipes take enable_activation_checkpointing and not activation_checkpointing, from what I see in the files? I was just intending to fix a typo here--I didn't think there would be BC issues. If there are BC issues I agree it would be better to not change.

Reply:
sorry, my bad :P